tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -291,13 +275,31 @@
           return;
         }
 
-        // Add a single root group element.
-        const margin = {top: 30, right: 10, bottom: 10, left: 10},
-              width = this.width - margin.left - margin.right,
-              height = this.height - margin.top - margin.bottom;
+        // We use the following algorithm for laying out our SVG:
+        // We compute a minimum size for the SVG based on the number columns
+        // and some margins. We set the svg "width" and "height" styles
+        // to "100%" so that it takes up the full area of its parent, but use
+        // "min-width" and "min-height", so that if the parent is too small
+        // the svg won't shrink down (it will overflow with scroll bars).
+        // If the parent is larger than the minimum size, we use the its viewBox
+        // attr to scale the contents to fit the larger size.
```

Comment:
If this was the intent, we can make the viewBox and lay out invariable to the size of the container and just use width/height CSS property on SVG to scale accordingly, right?

Reply:
I'm not sure I follow. Can you provide some code for your suggestion in a follow-up comment?